src/components/dialog/demoBasicUsage/script.js

```diff
@@ -4,24 +4,32 @@ angular.module('dialogDemo1', ['ngMaterial'])
   $scope.alert = '';
 
   $scope.showAlert = function(ev) {
-    $mdDialog.show(
-      $mdDialog.alert()
```

Comment:
I needed to supply a `parent` element. Dialogs must cover the whole screen, and because our docs have multiple `ng-app`s, they couldn't do that without an override. Can you supply a parent with the API-chain approach?